### PR TITLE
Add Dec 26 and Jan 02 holidays and other non working holidays of 2025

### PR DESCRIPTION
**Pull request by Sora19-paymongo**
Ticket: 
https://app.clickup.com/t/3870286/MM-5053

What:
- Add December 26 and January 02 holidays
- Add other regular and non-working holidays of 2025

### CODE REVIEW

**Comment by jamfrigillana** [comment]
Task linked: [MM-5053 [Core API, Payout API, Banking Calendar API]  Change Banking Calendar](https://app.clickup.com/t/3870286/MM-5053)

**Review by marc-suruiz-paymongo — COMMENTED** [review]
How about January 01, 2025?

File: lib/banking_calendar/data/bsp.yml [hunk 1]
```diff
@@ -109,5 +109,26 @@ bank_holidays:
   - 2024-12-08
   - 2024-12-24
   - 2024-12-25
+  - 2024-12-26
   - 2024-12-30
   - 2024-12-31
+  - 2025-01-01
+  - 2025-01-02
```
Comment (stephencina) [inline lib/banking_calendar/data/bsp.yml:116]: I don't think Jan 2 is a holiday. Can you double check?

File: lib/banking_calendar/data/bsp.yml [hunk 2]
```diff
@@ -109,5 +109,26 @@ bank_holidays:
...
+  - 2024-04-17
+  - 2024-04-18
+  - 2024-04-19
+  - 2025-05-01
+  - 2025-06-12
+  - 2025-08-21
+  - 2025-08-25
+  - 2025-10-31
+  - 2024-11-01
+  - 2024-11-08
```
Comment (stephencina) [inline lib/banking_calendar/data/bsp.yml]: Please double check this too.

**Review by stephencina — CHANGES_REQUESTED** [review]
Please double check. Thanks!

File: lib/banking_calendar/data/bsp.yml (hunk 2 above)
Comment (Sora19-paymongo) [reply lib/banking_calendar/data/bsp.yml]: Screenshot 2024-12-11 at 4 58 43 PM

File: lib/banking_calendar/data/bsp.yml (hunk 1 above)
Comment (marc-suruiz-paymongo) [reply lib/banking_calendar/data/bsp.yml:116]: Hmm, we make it a holiday so it won't generate a payout right?

**Review by marc-suruiz-paymongo — COMMENTED** [review]
Please update `banking_calendar` version. Thanks!

**Review by stephencina — APPROVED** [review]
Approving but please update banking calendar version.

File: lib/banking_calendar/data/bsp.yml [hunk 3]
```diff
@@ -109,5 +109,25 @@ bank_holidays:
...
   - 2024-12-24
   - 2024-12-25
+  - 2024-12-26
   - 2024-12-30
   - 2024-12-31
+  - 2025-01-01
+  - 2025-01-02
+  - 2025-01-29
+  - 2025-04-09
+  - 2024-04-17
```
Comment (marc-suruiz-paymongo) [inline lib/banking_calendar/data/bsp.yml]: Is this really 2024?

Comment (Sora19-paymongo) [reply lib/banking_calendar/data/bsp.yml]: revised to 2025

**Review by marc-suruiz-paymongo — APPROVED** [review]

**Pull request merged**